### PR TITLE
Eugenes mapbox

### PR DESCRIPTION
**Pull request by EugeneW9**

### CODE REVIEW

**Review by purgatorygates — APPROVED** [review]

File: client/src/components/Footer 2.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (ErnieAtLYD) [inline client/src/components/Footer 2.jsx:1]: Hey @EugeneW9 - talk to me about this file. This `Footer 2.jsx` looks exactly like `Footer.jsx`. Is this something you planned on purpose? If not, kill this file and submit a second commit to this branch.

File: client/src/components/Map.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { useRef, useEffect } from 'react';
+import mapboxgl from 'mapbox-gl';
+
+mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_ACCESS_TOKEN;
```
Comment (ErnieAtLYD) [inline client/src/components/Map.jsx:4]: Don't forget - if your teammate is going to also install this code on their machines, they'll need to know that they have to add `REACT_APP_MAPBOX_ACCESS_TOKEN` to their `.env` file. So make sure you got a file like `.env.sample` and add that. In a perfect world, you should call that out in a README.md as well when you write it out.

File: client/src/components/Map.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+const App = () => {
+  const mapContainerRef = useRef(null);
+
+  useEffect(() => {
+    const map = new mapboxgl.Map({
+      container: mapContainerRef.current,
+
+      style: 'mapbox://styles/mapbox/streets-v11',
+      center: [-104.9876, 39.7405],
```
Comment (ErnieAtLYD) [inline client/src/components/Map.jsx:14]: I'm assuming this is a placeholder, or the map is gonna be centered on that location forever.

**Review by ErnieAtLYD — APPROVED** [review]
I'll approve the PR BUT take a look at the comments and if any of that is true, go ahead and make those changes.

**Pull request closed without merging**